### PR TITLE
docs/networks: Add network config docs for Sepolia and Arbitrum Sepolia

### PR DESCRIPTION
**Pull request by fordN**
Adding network.md files for the new Testnet protocol instances.

### CODE REVIEW

File: docs/networks/arbitrum-sepolia.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+| --------------------------- | ----------------- |
+| Epoch length                | ~ 2h (554 blocks) |
+| Maximum allocation lifetime | ~ 9h (4 epochs)   |
+
+## Contracts & accounts
+
+| Name               | Address                                                                                                                        |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------ |
+| Graph Token (GRT ) | [`0xf8c05dCF59E8B28BFD5eed176C562bEbcfc7Ac04`](https://sepolia.etherscan.io/address/0xf8c05dCF59E8B28BFD5eed176C562bEbcfc7Ac04) |
+| Staking            | [`0x865365C425f3A593Ffe698D9c4E6707D14d51e08`](https://sepolia.etherscan.io/address/0x865365C425f3A593Ffe698D9c4E6707D14d51e08) |
```
Comment (Maikol) [inline docs/networks/arbitrum-sepolia.md:27]: Both addresses correspond to L2 versions so links should be on arbiscan.

File: docs/networks/arbitrum-sepolia.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+
+[comment]: <> (TODO: CHECK THESE)
+| Data Edge          | [`0xEFC8D47673777b899f2FB597C6FC0E87ecce98Cb`](https://sepolia.etherscan.io/address/0xEFC8D47673777b899f2FB597C6FC0E87ecce98Cb) |
```
Comment (Maikol) [inline docs/networks/arbitrum-sepolia.md:31]: This address is for the L1 `EventfulDataEdge`. The Arbitrum Sepolia address is `0x9b9402939133F27c6eba81a321dfBFa1feE6714E`

File: docs/networks/arbitrum-sepolia.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+| Block Oracle Owner | [`0xC9d59d6D2D43105357a1D9C15244751A4517f42C`](https://sepolia.etherscan.io/address/0xfA711DA0f9336f27E7B7483398cbd8F0880f259a) |
```
Comment (Maikol) [inline docs/networks/arbitrum-sepolia.md:32]: This address is for the Sepolia EBO, address for Arbitrum Sepolia is `0x76BC183A6d9AC1e4C5ccb27b7D46DDf0d2cc9868`

File: docs/networks/arbitrum-sepolia.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+| Parameter                   | Value             |
+| --------------------------- | ----------------- |
+| Epoch length                | ~ 2h (554 blocks) |
+| Maximum allocation lifetime | ~ 9h (4 epochs)   |
+
+## Contracts & accounts
+
+| Name               | Address                                                                                                                        |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------ |
+| Graph Token (GRT ) | [`0xf8c05dCF59E8B28BFD5eed176C562bEbcfc7Ac04`](https://arbiscan.io/address/0xf8c05dCF59E8B28BFD5eed176C562bEbcfc7Ac04) |
```
Comment (Maikol) [inline docs/networks/arbitrum-sepolia.md:26]: sorry if I wasn't clear, the link should be https://sepolia.arbiscan.io/address/0xf8c05dcf59e8b28bfd5eed176c562bebcfc7ac04

Same for other addresses on Arbitrum Sepolia.

**Review by Maikol — APPROVED** [review]

File: docs/networks/arbitrum-sepolia.md (hunk 4 above)
Comment (fordN) [reply docs/networks/arbitrum-sepolia.md:26]: Ah yes, of course. 🤦

**Pull request merged**